src/Documenter.jl

```diff
@@ -316,6 +317,18 @@ documentation. By default this value is set to `"master"`.
 **`devurl`** the folder that in-development version of the docs will be deployed.
 Defaults to `"dev"`.
 
+**`selector`** determines content and order of the resulting version selector in
+the generated html. The following entries are valied in the `selector` vector:
+ - `"v#"`: includes links to the latest documentation for each major release cycle
+   (i.e. `v1.1`, `v2.0`).
+ - `"v#.#"`: includes links to the latest documentation for each minor release cycle
+   (i.e. `v1.0`, `v1.1`, `v2.0`).
+ - `"v#.#.#"`: includes links to all released versions.
+ - `"v^"`: includes a link to the maximum version.
+ - A pair, e.g. `"first" => "second"`, which will put `"first"` in the selector,
+   with the same content as `"second"`. The second argument can be `"v^"`, to
+   point to the maximum version docs (as in e.g. `"stable" => "v^"`).
```

Comment:
Ditto as above w.r.t. URLs. It not only puts something into the selector, but also defines the URL you can access "second" from.